firebase_admin/dynamic_links.py

```diff
@@ -83,7 +79,7 @@ def _get_link_service(app):
 
 
 class LinkStats(object):
-    """The LinkStats object is returned by get_link_stats, it contains a list of EventStats"""
+    """The `LinkStats` object is returned by get_link_stats, it contains a list of `EventStats`"""
     def __init__(self, event_stats):
         if not isinstance(event_stats, (list, tuple)):
             raise ValueError('Invalid data argument: {0}. Must be a list or tuple'
```

Comment:
Your check in line 87 only validates the first entry. That is kind of pointless in a dynamically typed language. Either check all of them, or just assume they are all correct. For example, you could do:

```
if any([not isinstance(es, EventStats) for es in event_stats]):
  raise ValueError(...)
```